examples/zephyr/stream/README.md

```diff
@@ -1,11 +1,16 @@
-# Golioth LightDB Stream sample
+# Golioth Stream service sample
 
 ## Overview
 
-This LightDB Stream application demonstrates how to connect with Golioth
-and periodically send data to LightDB Stream. In this sample temperature
-measurements are sent to `/temp` LightDB Stream path. For platforms that
-do not have temperature sensor a value is generated from 20 up to 30.
+This Golioth Stream application demonstrates how to connect with Golioth
+and periodically send data to the Stream service. In this sample
+temperature measurements will be displayed on the `/temp` Stream
+path. For platforms that do not have a temperature sensor, a value is
+generated from 20 up to 30.
+
+> :information-source: To receive Stream data, a Pipeline must be
+> configured using the Golioth Web Console. This example is configured
+> to work with the default Pipeline.
```

Comment:
Was thinking about potentially removing the part about the default pipeline here and just putting the YAML for the pipeline directly in this README. That would be useful if you had previously deleted the default, or the default changes at some point in the future.
```suggestion
> :information-source: To receive Stream data, a Pipeline must be
> configured for the project on Golioth. This example is configured
> to work with the default Pipeline present for every project upon creation.
```